### ISSUE TITLE
Zsign + iOS 16 Support

### ISSUE THREAD

**Issue by QuixThe2nd** [zhlynn/zsign#257]
Has anyone confirmed whether or not zsign is working on iOS 16? I'm unable to install apps on iOS 16 and can't figure out whether it's an issue with Zsign or the profile itself. Any thoughts?
(How I'm creating profiles right now: [https://forum.starfiles.co/d/23-creating-an-ios-signing-certificate](https://forum.starfiles.co/d/23-creating-an-ios-signing-certificate))

**Comment by MartinLau7** [comment]
All is well for me

**Comment by QuixThe2nd** [comment]
What profile and certificate type are you using?

**Comment by QuixThe2nd** [comment]
> Is developer mode active on your device?

Yes it is.

It seems most third party app stores that use zsign aren't working on iOS 16 (even though some are). What are they doing differently?

**Comment by AKA2048** [comment]
If you use the online installation, you should check the itms-services protocol for plist issues

**Comment by QuixThe2nd** [comment]
Yeah I use itms-services. It works fine on iOS 12-15, just 16 that has issues, so i dont think the plist would have any issues, right?

**Comment by gyke69** [comment]
Hello,

Currently zsign work 50/50 for iOS 16, iOS 16 didn't change it's code-signature i fact Issue affects over-the-air installations. so this has to be checked/fixed by Apple.

Best Regards 
Arian

**Comment by iRayanKhan** [comment]
Hey @gyke69,

It seems that updated apps from 15 -> 16 work, but installing new apps just say "cannot install, please try again later."

Other signing services such appear to be working on 16 without a change, using Apple distribution certificate, while I tested with both iOS/Apple distribution and got the same "try again later" message. 

Apples own (non-App Store) apps seem to be able to update/install OTA.

**Comment by gyke69** [comment]
> Hey @gyke69,
> 
> It seems that updated apps from 15 -> 16 work, but installing new apps just say "cannot install, please try again later."
> 
> Other signing services such appear to be working on 16 without a change, using Apple distribution certificate, while I tested with both iOS/Apple distribution and got the same "try again later" message.
> 
> Apples own (non-App Store) apps seem to be able to update/install OTA.

Hello, 

We have to wait more and come to a conclusion. iOS 16 is pretty new so we definitely more reports.

Best Regards 
Arian

**Comment by QuixThe2nd** [comment]
Could it be a certificate issue? I've tried Apple Development and Apple Distribution with Ad Hoc profiles.

**Comment by Betamarrajr** [comment]
**Hope this will help for a future and permanent solution**

For all people that are having the “Unable to Install problem”  ( doesn’t not work for me on Duolingo ) but you can give it a try with other apps. 

Download sidelodly
Load the ipa file and select “remove extension” and export.

That will generate a new ipa file , sign and install that ipa.

Doing that I was able al least to install the app and to open it ( then I got something went wrong inside app loading lessons )

iPhone XS iOS 16b4

**Comment by yugu91** [comment]
> Hey @gyke69,
> 
> It seems that updated apps from 15 -> 16 work, but installing new apps just say "cannot install, please try again later."
> 
> Other signing services such appear to be working on 16 without a change, using Apple distribution certificate, while I tested with both iOS/Apple distribution and got the same "try again later" message.
> 
> Apples own (non-App Store) apps seem to be able to update/install OTA.

Yes, It's same for me, Have anything update?

**Comment by arm64x** [comment]
If using iOS 16, in plist file bundle id must be correct with bundle id of ipa file.

**Comment by yugu91** [comment]
> If using iOS 16, in plist file bundle id must be correct with bundle id of ipa file.

Thanks for sharing. Let me try!

**Comment by cychen-kw** [comment]
if using ios 16,  is there any way to run apps without developer mode?

**Comment by sugar1331** [comment]
zsign  在ios16之后的版本还能用吗，现在出现 无法安装此app因为无法验证其完整性
